test/topology_custom/random_failures/cluster_events.py

```diff
@@ -14,7 +14,7 @@
 
 from test.pylib.tablets import get_all_tablet_replicas
```

Comment:
> Now that we support suite subfolders, there is no
need to create an own suite for random_failures

Please put this explanation in the PR cover letter (with "random_failures" changed to "random_failures and tasks").